integration-tests/api-gateway/aws/env-variables.ts

```diff
@@ -0,0 +1,67 @@
+import { getSSMParameter } from "../../step-functions/aws/resources/ssm-param-helper";
+import { stackOutputs } from "../../step-functions/aws/resources/cloudformation-helper";
+
+export const CLIENT_ID = process.env.CLIENTID ?? "ipv-core-stub-aws-prod";
+export const CLIENT_URL =
+  process.env.CLIENT_URL ?? "https://cri.core.stubs.account.gov.uk";
+export const nino = "AA123456C";
+export const client_assertion_type =
+  "urn:ietf:params:oauth:client-assertion-type:jwt-bearer";
+export const grant_type = "authorization_code";
+export const environment = process.env.Environment || "localdev";
+export const claimSet = async () => {
```

Comment:
and maybe we should change the name of this the getClaimSet, since it is now a function.